protocol-tests/src/main/java/fixtures/constants/AutoRestSwaggerConstantServiceClient.java

```diff
@@ -0,0 +1,448 @@
+     * @throws HttpResponseException thrown if the request is rejected by server.
+     * @throws ClientAuthenticationException thrown if the request is rejected by server on status code 401.
+     * @throws ResourceNotFoundException thrown if the request is rejected by server on status code 404.
+     * @throws ResourceModifiedException thrown if the request is rejected by server on status code 409.
+     * @return the {@link Response}.
+     */
+    @Generated
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Response<Void> putModelAsStringRequiredOneValueNoDefaultWithResponse(RequestOptions requestOptions) {
+        return this.client.putModelAsStringRequiredOneValueNoDefaultWithResponse(requestOptions).block();
+    }
+
+    /**
+     * Puts constants to the testserver.
+     *
+     * <p><strong>Query Parameters</strong>
+     *
+     * <table border="1">
+     *     <caption>Query Parameters</caption>
+     *     <tr><th>Name</th><th>Type</th><th>Required</th><th>Description</th></tr>
+     *     <tr><td>input</td><td>String</td><td>Yes</td><td>The input parameter</td></tr>
+     * </table>
+     *
+     * @param requestOptions The options to configure the HTTP request before HTTP client sends it.
+     * @throws HttpResponseException thrown if the request is rejected by server.
+     * @throws ClientAuthenticationException thrown if the request is rejected by server on status code 401.
+     * @throws ResourceNotFoundException thrown if the request is rejected by server on status code 404.
+     * @throws ResourceModifiedException thrown if the request is rejected by server on status code 409.
+     * @return the {@link Response}.
+     */
+    @Generated
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Response<Void> putModelAsStringRequiredOneValueDefaultWithResponse(RequestOptions requestOptions) {
+        return this.client.putModelAsStringRequiredOneValueDefaultWithResponse(requestOptions).block();
+    }
+
+    /**
+     * Pass constants from the client to this function. Will pass in constant path, query, and header parameters.
+     *
+     * <p><strong>Query Parameters</strong>
+     *
+     * <table border="1">
+     *     <caption>Query Parameters</caption>
+     *     <tr><th>Name</th><th>Type</th><th>Required</th><th>Description</th></tr>
+     *     <tr><td>query-constant</td><td>int</td><td>Yes</td><td>Constant query property on the client that is a required parameter for operation 'constants_putClientConstants'</td></tr>
+     * </table>
+     *
+     * <p><strong>Header Parameters</strong>
+     *
+     * <table border="1">
+     *     <caption>Header Parameters</caption>
+     *     <tr><th>Name</th><th>Type</th><th>Required</th><th>Description</th></tr>
+     *     <tr><td>header-constant</td><td>boolean</td><td>Yes</td><td>Constant header property on the client that is a required parameter for operation 'constants_putClientConstants'</td></tr>
+     * </table>
```

Comment:
Currently, DPG seems not set the query/header if they are from client or is constant.

We should either document the constant value in Javadoc, or set it in code.